packages/rest/src/__tests__/unit/rest.server/rest.server.open-api-spec.unit.ts

```diff
@@ -375,6 +386,26 @@ describe('RestServer.getApiSpec()', () => {
     expect(spec.info).to.eql(EXPECTED_SPEC_INFO);
   });
 
+  context('OpenApiSpecOptions', () => {
+    it('disables consolidator if disableOASConsolidator is set to true', async () => {
+      const options: {rest: RestServerConfig} = {
+        rest: {openApiSpec: {disableOASConsolidator: true}},
+      };
+      options.rest = givenHttpServerConfig(options.rest);
+      app = new Application(options);
+      app.component(RestComponent);
+      server = await app.getServer(RestServer);
+      await server.start();
+
+      const enhancer = (await server.OASEnhancer.getEnhancerByName(
+        'consolidate',
+      )) as ConsolidationEnhancer;
+      expect(enhancer.isDisabled).to.be.true();
```

Comment:
Let's use `disabled` for consistency with other places that are already using this name. See e.g. here:

https://github.com/strongloop/loopback-next/blob/42e3f977181cae288c86ad1b227d6eba5c7e10df/packages/rest/src/rest.server.ts#L1068